code/modules/admin/player_panel.dm

```diff
@@ -297,15 +297,15 @@
 				var/mob/living/silicon/ai/A = M
 				if(A.client && A.eyeobj) // No point following clientless AI eyes
 					M_eyeUID = "[A.eyeobj.UID()]"
-			var/clientUID = M.client ? M.client.UID() : null
+			var/clientUID = M.client ? M.client.ckey : null
```

Comment:
Good old clientUID = client.ckey.

Reply:
It works! But yeah I changed the var names to match the value better